api/src/changeRequests/processChangeRequest.spec.ts

```diff
@@ -74,8 +84,8 @@ describe("processChangeRequest", () => {
                 name: "Xhoza",
             },
         };
-        await processChangeRequest(changeRequest1, accessMap, db);
-        const processResult = await processChangeRequest(changeRequest2, accessMap, db);
+        await processChangeRequest("", changeRequest1, accessMap, db);
```

Comment:
I understand we do it for testing, but shouldn't an empty user id be invalid?

Reply:
The user id is passed from the socket.io socket instance, and not from the user. I currently am not enforcing it as it is not a value that will be passed from the CMS / app